pkg/operator/ceph/provisioner/provisioner.go

```diff
@@ -151,15 +152,17 @@ func (p *RookVolumeProvisioner) createVolume(image, pool string, size int64) err
 func (p *RookVolumeProvisioner) Delete(volume *v1.PersistentVolume) error {
 	logger.Infof("Deleting volume %s", volume.Name)
 	if volume.Spec.PersistentVolumeSource.FlexVolume == nil {
-		return fmt.Errorf("Failed to delete rook block image %s/%s: %v", p.provConfig.pool, volume.Name, "PersistentVolume is not a FlexVolume")
+		return fmt.Errorf("Failed to delete rook block image %s: %v", volume.Name, "PersistentVolume is not a FlexVolume")
```

Comment:
i think it's useful to keep logging the pool name in all these log statements.  was there a compelling reason for removing that from the logging for you?

Reply:
The pool name might not necessarily be right given it is a residual value from the create operation which need not necessarily be right in this context. Figured it is best to avoid confusion logging something that might not be accurate.

Reply:
Now that we are persisting everything to the flexvolume options, we can't log the pool name here anyways because we haven't yet confirmed that we have the options to read from.  i'm OK with the pool name being removed from the logging here now.